spec/system/identity_spec.rb

```diff
@@ -25,6 +25,43 @@
     then_i_see_the_check_answers_page
   end
 
+  context "when sending the trn to the Identity API" do
+    it "redirects back to Get An Identity", vcr: true do
+      and_i_access_the_identity_endpoint_with_parameters
```

Comment:
Can we add an `alias_method` so this reads better?

```suggestion
      when_i_access_the_identity_endpoint_with_parameters
```